src/Provider/DoctrineMongoDbProvider.php

```diff
@@ -43,7 +50,7 @@ public function register(Container $container)
             $container['mongodbs.options'] = $tmp;
         });
 
-        $container['mongodbs'] = function ($container) {
+        $container['mongodbs'] = $container->factory(function ($container) {
```

Comment:
Why do you use a factory for this, using  $container['mongodbs'] should always return the same, or do i misunderstood something?